appeals/web/src/server/appeals/appeal-details/appellant-case/application-outcome/application-outcome.mapper.js

```diff
@@ -14,48 +15,37 @@ export const changeApplicationOutcomePage = (appealData, appellantCaseData, stor
 
 	/** @type {PageContent} */
 	const pageContent = {
-		title: 'Change the application outcome',
+		title: 'Change application decision outcome',
 		backLinkUrl: `/appeals-service/appeal-details/${appealData.appealId}/appellant-case`,
 		preHeading: `Appeal ${shortAppealReference}`,
-		heading: `What was the application decision?`,
+		heading: `Change application decision outcome`,
 		pageComponents: [
-			{
-				type: 'radios',
-				parameters: {
-					name: 'application-outcome',
-					idPrefix: 'application-outcome',
-					fieldSet: {
-						legend: {
-							text: `What was the application decision?`,
-							isPageHeading: false,
-							classes: 'govuk-fieldset__legend--1'
-						}
+			radiosInput({
+				name: 'application-outcome',
```

Comment:
no `idPrefix` being passed in?

Reply:
I'd omitted it because the `radios` component uses the `name` as the id by default, but as discussed it's better to specify it separately, so will update this

Reply:
Done!